tox.ini

```diff
@@ -1,4 +1,7 @@
 [tox]
-envlist = py27,py35,py36,py37,py38,py39,pypy
+envlist = py36,py37,py38,py39,pypy
```

Comment:
This would better be done using a GitHub actions matrix for multiple os/py version combinations.

Reply:
There is a GitHub Actions matrix, this was more for local testing. Tbh, I am lazy and  rely on the CI 99% of the time 😆
I can get rid of it.

Reply:
Same, I usually just push and pray ci passes. But even then there is netkos/act to run actions locally